.github/labeler.yml

```diff
@@ -0,0 +1,22 @@
+DB:
+  - src/**/*.sql
+
+CORE:
+  - src/**/*.cpp
+  - src/**/*.h
```

Comment:
shouldn't we exclude `src/server/scripts/**` ? or the category `Script` will make this excluded?